server/graphql/v2/mutation/VirtualCardMutations.ts

```diff
@@ -87,6 +87,70 @@ const virtualCardMutations = {
       return virtualCard;
     },
   },
+  createCardForCollective: {
+    description: 'Create new Stripe Virtual Card for existing hosted collective',
+    type: new GraphQLNonNull(VirtualCard),
+    args: {
+      name: {
+        type: GraphQLString,
+        description: 'Virtual Card name',
+      },
+      monthlyLimit: {
+        type: GraphQLInt,
+        description: 'Virtual Card monthly limit',
+      },
```

Comment:
We generally try to express amounts using the `Amount` and `AmountInput` objects.

Give clarity on wether the value is expressed in cents (x 100) or in plain currency.